rllib/algorithms/appo/torch/appo_torch_learner.py

```diff
@@ -35,10 +35,16 @@ class APPOTorchLearner(AppoLearner, TorchLearner):
     def compute_loss_per_module(
         self, module_id: str, batch: SampleBatch, fwd_out: Mapping[str, TensorType]
     ) -> TensorType:
-
         values = fwd_out[SampleBatch.VF_PREDS]
-        target_policy_dist = fwd_out[SampleBatch.ACTION_DIST]
-        old_target_policy_dist = fwd_out[OLD_ACTION_DIST_KEY]
+        action_dist_class = self.module[module_id].unwrapped().action_dist_cls
```

Comment:
the same api would be added to DDPTorchModule so the same code would work after Artur's PR is merged. 
https://github.com/ray-project/ray/pull/35085